src/test/java/com/wikia/webdriver/elements/mercury/components/Navigation.java

```diff
@@ -68,43 +64,24 @@ public JoinTodayPage clickOnSignInRegisterButton() {
     return new JoinTodayPage();
   }
 
-  public Navigation clickBackButton() {
-    Log.info("Go back to previous navigation level");
-    wait.forElementClickable(backButton);
-    backButton.click();
-
-    return this;
-  }
-
-  public Navigation clickExploreWikiHeader(Skin fromSkin) {
-    Log.info("Click 'Explore Wiki' header");
-    wait.forElementClickable(exploreWikiHeader);
-
-    exploreWikiHeader.click();
-
-    // Mobile wiki opens the main page using AJAX, Mercury reloads the page and opens Mobile Wiki
-    if (fromSkin == Skin.MOBILE_WIKI) {
-      waitForPageReload();
-    } else {
-      new SkinHelper(driver).isSkin(Skin.DISCUSSIONS);
-    }
+  public Navigation closeNavigation() {
+    Log.info("Close sub-menu");
+    wait.forElementClickable(closeButton);
+    closeButton.click();
 
     return this;
   }
 
-  public Navigation closeSubMenu() {
-    Log.info("Close sub-menu");
-    wait.forElementClickable(backButton);
-    backButton.click();
-
+  public Navigation clickWikisMenuLink() {
+    wait.forElementClickable(wikisDropdownMenu);
+    this.hover(wikisDropdownMenu);
     return this;
   }
-
   public Navigation openSubMenu(int index) {
     Log.info("Open sub-menu no.: " + index);
-    WebElement wikiMenuLink = subMenuLinks.get(index);
-    wait.forElementClickable(wikiMenuLink);
-    wikiMenuLink.click();
+    //WebElement wikiMenuLink = subMenuLinks.get(index);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)